client/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import Carousel from './components/carousel.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      photoArray: [],
+      displayCarousel: false,
+    };
+  }
+
+  retrievePhotos(restaurantID) {
+    const options = {
+      method: 'get',
+      url: `api/photos/${restaurantID}`,
+      failure: () => {
+        console.log('Request failed');
+      },
+      complete: (res) => {
+        this.setState({ photoArray: res.responseJSON });
+      },
+    };
+    $.ajax(options);
```

Comment:
This is random, but I was just reflecting about this. I highly recommend looking into axios instead of jquery to make calls to the api from react. It's promisified already and it can be helpful when you need to invoke a re-render. I know 'complete' does the same, but I find axios way more intuitive than jquery to ensure operations happen in a specific order and errors are handled correctly.

Reply:
Double down Cinzia. Completely agree. I've also read that we shouldn't be using JQuery + React together and Axios might be lighter. Not that I know, but just a thought.